lib/routemaster/jobs/cache_and_sweep.rb

```diff
@@ -6,7 +6,8 @@ module Jobs
     # Caches a URL using {Cache} and sweeps the dirty map if successful.
     # Busts the cache if the resource was deleted.
     class CacheAndSweep
-      def perform(url)
+      def perform(url, source_peer = nil)
```

Comment:
Let's pass it as an optional hash.

```ruby
def perform(url, options = {})
  @source_peer = options[:source_peer]
...
```

In fact, you might want to just set cache.
```
def perform(url, options = {})
  @cache = options[:cache] || Cache.new
```

Reply:
Also, please add details of change to CHANGELOG for each PR :slightly_smiling_face:

Reply:
It would have been tricky to pass the `cache` through as this is in a sidekiq job.
We went with an options hash (specific to `client_options`, which we can then use to instantiate the `Routemaster::Cache`).